config/action.d/freebsd-ipfw.conf

```diff
@@ -0,0 +1,99 @@
+# Fail2Ban configuration file
+#
+# This file was created to provide ipfw with a configuration for FreeBSD
+# as flexible as possible
+# Author:	Nilton J. Rizzo
+#
+#
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = 
+
+
+# Option:  actionstop
+# Notes.:  command executed once at the end of Fail2Ban
+# Values:  CMD
+#
+actionstop =  ipfw delete <rulenumber>
+
+
+# Option:  actioncheck
+# Notes.:  command executed once before each actionban command
+# Values:  CMD
+#
+actioncheck = 
+
+
+# Option:  actionban
+# Notes.:  command executed when banning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    See jail.conf(5) man page
+# Values:  CMD
+#
+#actionban = ipfw add <rulenumber> <blocktype> <proto> from <ip> to <localhost> dst-port <port> in via <nic> 
+actionban = ipfw add <rulenumber> <blocktype> <proto> from <ip> to <localhost> <port>
+
+
+# Option:  actionunban
+# Notes.:  command executed when unbanning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    See jail.conf(5) man page
+# Values:  CMD
+#
+actionunban = ipfw delete `ipfw list | grep -i "[^0-9]<ip>[^0-9]" | awk '{print $1;}'`
+
+[Init]
+
+# Option:  port
+# Notes.:  specifies port to monitor
+# Values:  [ NUM | STRING ]
+#
+port = ssh
+
+# Option:  localhost
+# Notes.:  the local IP address of the network interface
+# Values:  IP
+#
+#localhost = me # first nic on system
+localhost = 127.0.0.1
+
```

Comment:
Have you tested this on your system? 
I suspect your firewall rules will not work.
`ipfw add <rulenumber> <blocktype> <proto> from <ip> to <localhost> <port>` from `actionban` will result in
`10000           deny ip from <123.456.789.0> to 127.0.0.1 22`
No external host will try to reach you at 127.0.0.1

I see that you have `localhost = me` commented out. (You uncommented it in the subsequent change.)
But even if the host has multiple IPs why would someone need to fiddle with only one IP, instead of blocking all of them (see my comment below regarding the interface name).